sdk/appconfiguration/app-configuration/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Other Changes
 
+- Migrated from using `@azure-tools/test-recorder` to depend on the newer v2 of the recorder. [#23520](https://github.com/Azure/azure-sdk-for-js/pull/23520). See [Azure @azure-tools/test-recorder library for JavaScript](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/recorder/README.md) for more information on the new test recorder.
```

Comment:
This is not needed for our users.
In the changelog, we only mention changes that are meaningful to some extent to our users.